### PR TITLE
[#39] [Chore] Setup CD pineline

### LINKED ISSUE #39: [Chore] setup CD pineline
## Why

- No manual execution of releases
 
## Who Benefits?
- CD pipeline is run on Github Action
- CD pipeline run to build a staging version and upload to Firebase Distribution

### PR DESCRIPTION
**Pull request by nkhanh44**
#39 

## What happened

- App need to be distributed on Firebase automatically.
 
## Insight

- Configured CD pineline
- Set up firebase app for production and staging environments.
 
## Proof Of Work

- Distributed by fastlane:
Screen Shot 2022-08-19 at 13 22 36

- Firebase's build file:
Screen Shot 2022-08-19 at 13 33 29

- Match Ad-hoc on fastlane:
Screen Shot 2022-08-22 at 11 07 31

- Github Action run:
https://github.com/nkhanh44/ic-survey-ios/actions/runs/2902877806

### CODE REVIEW

File: fastlane/Matchfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-git_url("git@github.com:{organization}/{repo}.git")
+git_url("git@github.com:nkhanh44/ic-survey-ios.git")
```
Comment (blyscuit) [inline fastlane/Matchfile]: Normally we create a new private match repo and use it alongside the project. Not sure how it works when using the same project to match 🤔 
Also using same project for match will cause the repo to update itself from time to time (when running match), which is an undesirable scenario.

**Review by blyscuit — COMMENTED** [review]
Will you use Github Actions (or other tool) to run this pipeline?

**Comment by nkhanh44** [comment]
@blyscuit  Yes, I will use Github Actions to run this pipeline 😄

File: fastlane/Matchfile (hunk 1 above)
Comment (nkhanh44) [reply fastlane/Matchfile]: You're right, i will create a new private repo 🙏 , thank you.

**Review by ducbm051291 — APPROVED** [review]
LGTM

**Review by blyscuit — APPROVED** [review]

**Pull request merged**